Library/Homebrew/tap.rb

```diff
@@ -31,7 +31,7 @@ def self.fetch(*args)
     repo = repo.strip_prefix "homebrew-"
 
     if user == "Homebrew" && repo == "homebrew" ||
-        user == "Linuxbrew" && repo == "linuxbrew"
+        repo == "linuxbrew"
```

Comment:
Yes, I believe this is incorrect, I only wanted to test it. In my case I get `user == "DoomHammer" && `repo == "linuxbrew"` when running `test-bot` on my PR on Travis. Could you help me find out what is wrong here?